### ISSUE TITLE
Make Miners Less Squishy!

### ISSUE THREAD

**Issue by BellePullman** [ldtteam/minecolonies#6075]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.3
- MineColonies Version: 0.13.440
- Structurize Version: 0.13.96
- Related Mods and their Versions:

## Expected behavior

Miners can work productively; incidents are the exception not the rule.
## Actual behavior

I have a well established level 4-5 town with three miners, each hut built in a different style.  I am losing at least one miner to mobs every in-game day.  Citizens mourning a death every in-game day is a huge hit on development, the lack of mining means shortages of iron, coal, and cobble etc.  The miner basically does not work.  I have guards stationed on the same level of each mine yet every time I reach the scene of death, the guard is in entirely the wrong area, even when it is a small mineshaft with minimal nodes.

## Steps to reproduce the issue

1. Set the miner to work
2. Set a guard to guard on the same level
3. Switch to Spectator Mode to quickly find what killed the miner this time.

### Logs

- latest.log: 
- crashlog: 

### Notes

Suggestions to fix the problem:
1) The ability for a Guard to actually follow the Miner has been discussed, and would help for sure.  But I do not think it alone would be enough - if a miner and a guard walk into a mob filled room, the guard will not take out all the hostile mobs fast enough to avoid the miner taking fatal damage.
2) Buff miners in the code - I'm not a programmer but for example, the Tavern Visitors are hard to kill, give Miners a similar buff.
3) Miner's Helmets - again similar to the visitors wearing boots, allow miners to equip helmets, that can be enchanted at higher levels.
4) Increase light levels in the mine schematics.  As it stands even a fully completed mine node may not reach mob spawn-proof light levels.  Significantly increasing the light levels would allow more light to spill into the problem areas.
5) Change the way Miners mine.  As it stands they start at the top back of the node, mining out the top layer and working downwards.  This creates an unlit chamber.  the miner leaves for more supplies, or night falls, mobs spawn, the miner returns and dies.  Instead miners could work from the front to the back, allowing light from the existing mine to spill into the new area.  Also, miners could be required to have all the materials in their inventory before starting work, and not leave the node until the lighting is installed.  This would be more achievable if the lighting was not a separate second stage, but torches placed as their spaces are exposed.
6) Cowardly Miners - they who fight and run away, live to mine another day?  If the miner was more aware of the danger and ran away faster, they might not die quite so very often.
---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.
